website/blog/2024-05-24-Agent/index.mdx

```diff
@@ -0,0 +1,142 @@
+One can start from relatively simple agents, compose them together and make them interact
+in a good way. Then they would exhibit a higher level of intelligence.
+
+We don't have a good single agent that
+can do everything we want yet. And why is that? It could be because we haven't figured out the right
+way of composing the multi-agent to build this powerful single agent. But firstly, we need to have the
+framework that allows easy experimentation of these different ways of combining models and agents. My own experience is that if people practise using multi agents to solve a problem, they often reach a solution faster.
+I have high hopes that they can figure out a robust way of building a single agent using this way.
+Otherwise there are just too many possibilities. There are too many ways to build this single agent. If
+every time you rewrite the entire system, it's not going to be easy to maintain and make progress.
+So that's why this modularity from a programming point of view is very useful.
+
+On the other hand, we
+don't have to stop there. We can think about multi-agent system as a way to multiply the
+power of a single agent. We can connect them with other agents to accomplish bigger goals.
+
+### Benefits of multi-agents
+
+There are at least two types of applications that benefit from multi-agent systems.
+
+- Single-agent interface.
+Developers often find that they need to extend the system with different capabilities, tools, etc. And if
+they implement that single agent interface with a multi-agent architecture, they can often increase the capability to
+handle more complex tasks or improve the quality of the response. One example is complex data analytics.
+It often requires agents of different roles to solve a task. Some agents are good at retrieving the data and
+presenting to others. Some other agents are good at running deep analytics and providing insights. We can also have agents which can do critics and suggest more actions. Or agents that can do planning, and so on.
+Usually, to accomplish a complex task, one can build these agents with different roles.
+> If you don't know about Chi Wang and Microsoft Research's work, please check it out. I want to give a real world production use case for Skypoint AI platform client Tabor AI https://tabor.ai AI Copilot for Medicare brokers - selecting a health plan every year for seniors (65 million seniors have to do this every year in the US) is a cumbersome and frustrating task. This process took hours to complete by human research, now with AI agents 5 to 10 minutes without compromising on quality or accuracy of the results. It's fun to see agents doing retail shopping etc. where accuracy is not that mission critical. AI in regulated industries like healthcare, public sector, financial services is a different beast, this is Skypoint AI platform (AIP) focus.
```

Comment:
The inline quotes are bit distracting... have them at the end or together in the same place is better. There is also no mention of the quote and context from the article itself.

Reply:
Agreed... is Tabor AI providing a single-agent interface? If so, perhaps start with "As an example of a real-world production use case for single-agent interfaces, Skypoint AI's platform, Tabor AI..."

If not, then perhaps at the end of the article in a separate section.